nntrainer/app_context.cpp

```diff
@@ -255,6 +256,19 @@ static void add_default_object(AppContext &ac) {
   ac.registerFactory(nntrainer::createLayer<PermuteLayer>, PermuteLayer::type,
                      LayerType::LAYER_PERMUTE);
 
+  /** register losses */
+  ac.registerFactory(nntrainer::createLayer<MSELossLayer>, MSELossLayer::type,
+                     LayerType::LAYER_LOSS_MSE);
+  ac.registerFactory(nntrainer::createLayer<CrossEntropyLossLayer>,
```

Comment:
Wonder if we need to keep CrossEntropyLossLayer

Reply:
Maybe we need as cross entropy can be the last layer

Reply:
Cross is opened with the APIs, so it kept. This layer can be created and added, but cant be finalized.
This must be merged with appropriate activations for it work.